gluster-ansible.spec

```diff
@@ -42,6 +42,9 @@ install -p -m 644 README.md LICENSE %{buildroot}/%{docdir}
 %license LICENSE
 
 %changelog
+* Fri Apr 01 2022 Sandro Bonazzola <sbonazzo@redhat.com> - 1.0.5-26
```

Comment:
it should be 1.0.5-27

Reply:
Fixed in https://github.com/gluster/gluster-ansible/pull/137/commits/ee1728bc66295af57f673c0c6f9c352ac8186f04